config/settings.yml

```diff
@@ -415,7 +415,6 @@ evss:
   mock_common_service: <%= ENV['evss__mock_common_service'] %>
   mock_disabilities_form: <%= ENV['evss__mock_disabilities_form'] %>
   mock_gi_bill_status: <%= ENV['evss__mock_gi_bill_status'] %>
-  mock_itf: <%= ENV['evss__mock_itf'] %>
```

Comment:
➡️ Follow these instructions post merge and deploy: https://depo-platform-documentation.scrollhelp.site/developer-docs/settings-and-parameter-store#SettingsandParameterStore-DeleteaSetting